pydatalab/pydatalab/routes/v0_1/collections.py

```diff
@@ -325,3 +326,51 @@ def search_collections():
     ]
 
     return jsonify({"status": "success", "data": list(cursor)}), 200
+
+
+@COLLECTIONS.route("/collections/<collection_id>", methods=["POST"])
+def add_items_to_collection(collection_id):
+    data = request.get_json()
+    refcodes = data.get("data", {}).get("refcodes", [])
+
+    collection = flask_mongo.db.collections.find_one({"_id": ObjectId(collection_id)})
+    if not collection:
+        return jsonify({"error": "Collection not found"}), 404
+
+    if not refcodes:
```

Comment:
Can we add a couple of Python tests for this functionality too? Ideally this could extend the item graph tests